mypy/waiter.py

```diff
@@ -62,7 +62,7 @@ def status(self) -> Optional[int]:
     def read_output(self) -> str:
         with open(self.outfile.name, 'rb') as file:
             # Assume it's ascii to avoid unicode headaches (and portability issues).
-            return file.read().decode('ascii')
+            return file.read().decode('utf-8')
```

Comment:
Hm, the ASCII was probably intentional (so we won't have mystery Unicode characters in tests). What prompted this change?